acceptance/tests/agent/agent_can_reconnect_after_master_failure.rb

```diff
@@ -0,0 +1,49 @@
+test_name 'C97323: agents can reconnect after master crash/restart' do
+
+  logfiles = {}
+  agents.each do |agent|
+    logfiles[agent.hostname] = agent.tmpfile(File.basename(__FILE__, '.*'))
+  end
+
+  teardown do
+    step 'stop puppet agent' do
+      on(agents, puppet_resource("service puppet ensure=stopped"))
+    end
+  end
+
+  step 'start puppet master' do
+    with_puppet_running_on(master, {}) do
+      agents.each do |agent|
+        step 'start puppet agent at rapid runinterval' do
+          on(agent, puppet("agent --server #{master.hostname} --runinterval 10 " +
+                           "--logdest #{logfiles[agent.hostname]} " +
+                           "--user #{agent['user']}"))
```

Comment:
I think you can instead manually call to SC and pass the params (at least that's what our docs say ;0):

`sc start --server #{master.hostname} --runinterval 10 --logdest #{logfiles[agent.hostname]} --user #{agent['user']}`

Just keep in mind `#{logfiles[agent.hostname]}` should be a Windows style path.